Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:3.4
+
+RUN apt-get update && apt-get install -y valgrind
```

Comment:
`--no-install-recommends` and `--no-install-suggests`

Reply:
thanks! Fixed in [38c47c2](https://github.com/DataDog/libddwaf-rb/pull/78/commits/38c47c25b388b39681031e77e0aea79877470beb) (also for the Github Action)